visidata/loaders/xlsx.py

```diff
@@ -203,8 +203,11 @@ def save_xls(vd, p, *sheets):
 
 @XlsxSheet.api
 def colorize_xlsx_cell(sheet, col, row):
-    fg = getattrdeep(row, col.column_letter+'.font.color', None)
-    bg = getattrdeep(row, col.column_letter+'.fill.start_color', None)
+    if hasattr(col, 'column_letter'):
+        fg = getattrdeep(row, col.column_letter+'.font.color', None)
+        bg = getattrdeep(row, col.column_letter+'.fill.start_color', None)
+    else:
+        fg = bg = None
```

Comment:
Let's early return instead, to not call `xlsx_color_to_xterm256` unnecessarily, and to reduce the size of the diff.

Reply:
Oh right. I didn't even look at the rest of the function. I wasn't sure whether to go with `None` or `''`. It seems both are fine for now.